core/quantity/src/Interfaces.ts

```diff
@@ -38,6 +38,8 @@ export interface UnitConversionSpec {
   name: string;
   /** The default label that is used to display unit */
   label: string;
+  /** Unit system name, used to when finding preferred parse unit */
+  system: string;
```

Comment:
We should probably add a note in the NextVersion.md about this as it is a breaking change even though it's a `alpha` API.

Reply:
Added note to breaking change section.